common/web/types/src/keyman-touch-layout/keyman-touch-layout-file.ts

```diff
@@ -52,8 +52,9 @@ export const PRIVATE_USE_IDS = [
    * Private-use identifiers used by KeymanWeb for the default multitap-into-caps-layer key
```

Comment:
These have grown into more than just multitap-into-caps...  Maybe multitap-shift-into other layers?

Reply:
Not really... it's still pretty accurate.  These only provide for caps-related multitap defaults.